vxyowsup/whatsapp.py

```diff
@@ -154,8 +154,8 @@ def onMessage(self, messageProtocolEntity):
             messageProtocolEntity.getParticipant())
         self.toLower(receipt)
 
-        print "You have received a message, and thusly sent a receipt"
-        print "You are now sending a reply"
+        log.debug('You have received a message, and thusly sent a receipt')
+        # log.debug("You are now sending a reply")
         # self.send_to_human('iiii', from_address + '@s.whatsapp.net', "this transport's gone rogue")
```

Comment:
Maybe we can add an `echo_to` flag to the config that bounces messages to a specified MSISDN? That would allow us to have an official testing mode for debugging? E.g.
```yaml
# config.yaml
echo_to: +5551234
```
and then
```python
# whatsapp.py
if self.transport.config.echo_to:
    self.send_to_human(self.transport.config.echo_to + '@s.whatsapp.net', from_address + '@s.whatsap.net', content)
```
Thoughts?